docs/ipv6.md

```diff
@@ -0,0 +1,170 @@
+
+## Limitations
+
+Currently, this setup makes it possible to `ping6` one pod from another. We look forward to using this laboratory to explore the other possibilities and scenarios herein, however this pod-to-pod `ping6` is the baseline functionality from which to start.
+
+## Requirements
+
+This also assumes that you have:
+
+* A machine for running Ansible (like your workstation) and [have Ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html).
+* Ansible 2.4 or later (necessary to support `get_url` with IPv6 enabled machines)
+* A host capable of running virtual machines, and is running CentOS 7.
+* A clone of this repository.
+
+This scenario disables the "bridged networking" feature we often use and instead uses NAT'ed libvirt virtual machines. 
+
+You may have to disable GRO ([generic receive offload](https://en.wikipedia.org/wiki/Large_receive_offload)) for the NICs on the virtualization host (if you're using one).
+
+An example of doing so is:
+
+```
+ethtool -K em3 gro off
+```
+
+## Inventory and variable setup
+
+Firstly, let's look at an inventory and variable overrides to use.
+
+Here's the initially used inventory, which only really cares about the `virthost`. These examples place the inventory file @ `inventory/your.virthost.inventory`. You'll need to modify the location of the host to match your environment.
+
+```
+the_virthost ansible_host=192.168.1.119 ansible_ssh_user=root
+
+[virthost]
+the_virthost
+```
+
+And the overrides which are based on the examples @ `./inventory/examples/virthost/virthost-ipv6.inventory.yml`. In these examples this set of extra variables is located @ `./inventory/extravars.yml`:
+
+```
+bridge_networking: false
+virtual_machines:
+  - name: kube-master
+    node_type: master
+  - name: kube-node-1
+    node_type: nodes
+  - name: kube-node-2
+    node_type: nodes
+  - name: kube-nat64-dns64
+    node_type: other
+ipv6: true
+```
+
+
+## Spinning up and access virtual machines
+
+Perform a run of the `virthost-setup.yml` playbook, using the previously mentioned extra variables for override, and an inventory which references the 
+
+```
+ansible-playbook -i inventory/your.virthost.inventory -e "@./inventory/extravars.yml" virthost-setup.yml
+```
+
+This will produce an inventory file in the local clone of this repo @ `./inventory/vms.local.generated`.
+
+And I have to modify the generated inventory to jump host, as I run ansible from my laptop.
```

Comment:
s/ansible/Ansible in full document

Reply:
Thanks, actually meant to remove that line from the raw notes.